src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -56,7 +56,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.2.1" />
```

Comment:
Not for the PR - I wonder that we don't update this to 3.2.0 https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/3.2.0

Reply:
Why? That would be a downgrade.

Reply:
I think he meant: https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/3.3.1-beta3-final
![image](https://user-images.githubusercontent.com/10873629/64376783-e482a500-cfdd-11e9-815f-c1e749073c14.png)

Reply:
Yes, sorry.